package.json

```diff
@@ -24,9 +24,13 @@
     "automated:watchMe:xunit:win": "suitest automated --log-dir=e2e mocha --reporter xunit --reporter-options output=./reports/xunit/%SUITEST_CHILD_PROCESS:~0,36%.xml,suiteName=DEVICE_ID-%SUITEST_CHILD_PROCESS:~0,36% --no-timeouts --exit \"./watchMeTests/*.test.js\"",
     "automated:watchMe:tap": "suitest automated --log-dir=e2e mocha --reporter tap  --no-timeouts --exit \"./watchMeTests/*.test.js\"",
     "automated:watchMe:mochawesome": "suitest automated --log-dir=e2e mocha --reporter mochawesome --reporter-options reportTitle=DEVICE_ID-'\"${SUITEST_CHILD_PROCESS%%|*}\"',reportFilename='\"${SUITEST_CHILD_PROCESS%%|*}\"',reportDir='./reports/html',autoOpen=true  --no-timeouts --exit \"./watchMeTests/*.test.js\"",
-    "automated:watchMe:mochawesome:win": "suitest automated --log-dir=e2e mocha --reporter mochawesome --reporter-options reportTitle=DEVICE_ID-%SUITEST_CHILD_PROCESS:~0,36%,reportFilename=%SUITEST_CHILD_PROCESS:~0,36%,reportDir=/reports/html,autoOpen=true  --no-timeouts --exit \"./watchMeTests/*.test.js\""
+    "automated:watchMe:mochawesome:win": "suitest automated --log-dir=e2e mocha --reporter mochawesome --reporter-options reportTitle=DEVICE_ID-%SUITEST_CHILD_PROCESS:~0,36%,reportFilename=%SUITEST_CHILD_PROCESS:~0,36%,reportDir=/reports/html,autoOpen=true  --no-timeouts --exit \"./watchMeTests/*.test.js\"",
+    "interactive:slack-reporter:win": "suitest interactive mocha --reporter mocha-ci-slack-reporter --reporter-options username=username,url=slack_incoming_webhooks_url,channel=#mocha-reports,testTitle=DEVICE_ID-%SUITEST_CHILD_PROCESS:~0,36% --no-timeouts --exit \"./test/dummy.test.js\"",
```

Comment:
Need remove this command or, add unix like alternative.
@AntonSBilan what do you think?

Reply:
As for me I'd remove it as, there is not so much reasons why one may use it in interactive mode and we don't have interactive commands for other reporters

Reply:
Could you remove it please?

Reply:
done